lib/features/push_notification/data/datasource_impl/remote_web_socket_datasource_impl.dart

```diff
@@ -32,6 +33,8 @@ class RemoteWebSocketDatasourceImpl implements WebSocketDatasource {
     Session session,
     AccountId accountId,
   ) async* {
+    if (PlatformInfo.isMobile) throw WebSocketPushNotSupportedException();
```

Comment:
No need to check here. We just inject Web Socket dependencies for web.

Reply:
Removed